cfripper/rules/base_rules.py

```diff
@@ -15,6 +15,7 @@
 
 
 class Rule(ABC):
+    ACTIONABILITY = True
```

Comment:
I'd rename the field to: ACTIONABLE instead of ACTIONABILITY
Since it'll be either actionable or non-actionable. Actionability sounds "weird"
```suggestion
    ACTIONABLE = True
```

Reply:
Do we also want to include the new property in the config or not?

Reply:
I wasn't sure of it when I added my other comment. Now that I think of it, we can safe proof it, and add it to the config with the same value. This way, if some user/dependency of cfripper needs to override the value, it can be done safely in their config 👍